kotlin-reference-server/src/main/kotlin/com/example/sep24/Sep24Util.kt

```diff
@@ -0,0 +1,116 @@
+package com.example.sep24
+
+import com.example.data.PatchTransactionRecord
+import com.example.data.PatchTransactionsRequest
+import com.example.data.Transaction
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import java.math.BigDecimal
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.delay
+import kotlinx.serialization.json.Json
+import mu.KotlinLogging
+import org.stellar.sdk.*
+import org.stellar.sdk.responses.TransactionResponse
+
+object Sep24Util {
+  private val log = KotlinLogging.logger {}
+
+  // TODO: make parameters in this class configurable
+  private val myKey =
+    System.getenv("STELLAR_KEY") ?: "SDYGC4TW5HHR5JA6CB2XLTTBF2DZRH2KDPBDPV3D5TXM6GF7FBPRZF3I"
+  internal val myKeyPair = KeyPair.fromSecretSeed(myKey)
+
+  val client = HttpClient {
+    install(ContentNegotiation) { json(Json { ignoreUnknownKeys = true }) }
+  }
+
+  val baseUrl = "http://localhost:8080"
```

Comment:
We should probably make this configurable via environment variable as well

Reply:
I planned to do it later (in next PR for ref server) to get this PR merged ASAP to avoid big merge conflicts

Reply:
Its up to you. If there are other PRs in-progress that will cause conflicts then sure